app/components/agenda/agenda-header/agenda-actions.js

```diff
@@ -333,6 +333,42 @@ export default class AgendaAgendaHeaderAgendaActions extends Component {
     this.args.didApproveAgendaitems();
   }
 
+  @action
+  async markDecisionsForSigning() {
+    const agenda = this.args.currentAgenda;
+    const agendaitems = await agenda.agendaitems;
+    const reports = await this.store.queryAll('report', {
+      'filter[decision-activity][treatment][agendaitems][:id:]': agendaitems
+        .map((x) => x.id)
+        .join(','),
+    });
+    const loadingToast = this.toaster.loading(
```

Comment:
**suggestion (non-blocking)**: the toasts could be a little more descriptive for edge cases

If there are no reports we might want to show a toast that nothing has been marked instead. We might also want to show a count of reports being marked (not all agendaitems might have a report). Nothing urgent so no need to change for this PR, if users mention it we can make a ticket for it down the line.